libs/core/calendar/calendar-views/calendar-month-view/calendar-month-view.component.ts

```diff
@@ -82,6 +99,11 @@ export class CalendarMonthViewComponent<D> implements OnInit, OnChanges, Focusab
     /** @hidden */
     private _initiated = false;
 
+    /** @hidden */
+    private _selectedRangeDate: DateRange<D>;
```

Comment:
same thing, why selectedRangeDate

Reply:
@dpavlenishvili To store the selectedDate locally in the component and to apply hover styles on the range